src/CMakeLists.txt

```diff
@@ -355,6 +355,15 @@ if(BUILD_UTIL_CHAINSTATE)
   add_executable(bitcoin-chainstate
     bitcoin-chainstate.cpp
   )
+  # TODO: The `SKIP_BUILD_RPATH` property setting can be deleted
```

Comment:
I don't understand this. Don't we always want the BUILD_RPATH for bitcoin-chainstate so that it works with a shared kernel in the build tree? Then it's stripped for the INSTALL_RPATH binary and everyone's happy?

Reply:
> Don't we always want the BUILD_RPATH for bitcoin-chainstate so that it works with a shared kernel in the build tree?

We do, of course.

I mean, in the future, this code: https://github.com/hebasto/bitcoin/blob/02d9f7faeb2cb4cb1e93f28008f109e533807b9a/CMakeLists.txt#L606-L607 can be just:
```cmake
set(CMAKE_SKIP_INSTALL_RPATH TRUE) 
```
which effectively makes `set_target_properties(bitcoin-chainstate PROPERTIES SKIP_BUILD_RPATH OFF)` redundant.

Or do you mean always keep it for the purpose of explicitness.